rest/routing.go

```diff
@@ -185,6 +185,9 @@ func CreateAdminRouter(sc *ServerContext) *mux.Router {
 	dbr.Handle("/_user/{name}",
 		makeHandler(sc, adminPrivs, []Permission{PermWritePrincipal}, nil, (*handler).deleteUser)).Methods("DELETE")
 
+	dbr.Handle("/_user/{name}/all_channels",
+		makeHandler(sc, adminPrivs, []Permission{PermReadPrincipal}, nil, (*handler).handleGetAllChannels)).Methods("GET")
```

Comment:
As discussed, make decision on new port vs. subrouter and move this endpoint to reflect that decision.